flows/codelingo/rewrite/rewrite/cli.go

```diff
@@ -50,6 +50,10 @@ var CLIApp = &flowutil.CLIApp{
 				Name:  "debug",
 				Usage: "Display debug messages",
 			},
+			cli.StringFlag{
+				Name:  "github-comments, g",
```

Comment:
Is the json format github specific? If not, it'd be better to call this `dump-comments`. We can then reuse the same dump for other platforms.